### PR TITLE
[FX-5686] Migrate PageContent to TailwindCSS

### PR DESCRIPTION
**Pull request by ruslan-sed**
[FX-5686]

### How to test

- [Temploy](https://picasso.toptal.net/fx-5686-migrate-pagecontent)

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Double check if `picasso-tailwind-merge` requires major update (check its `README.md`)
- ~Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)~
- ~Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core~
- ~Annotate all `props` in component with documentation~
- ~Create `examples` for component~
- [x] Ensure that deployed demo has expected results and good examples
- ~Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).~
- [x] Self reviewed
- ~Covered with tests ([visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md) included)~

**Breaking change**

- ~codemod is created and showcased in the changeset~
- ~test alpha package of Picasso in StaffPortal~

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-5687]: https://toptal-core.atlassian.net/browse/FX-5687?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[FX-5686]: https://toptal-core.atlassian.net/browse/FX-5686?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ruslan-sed** [comment]
@toptal-anvil ping reviewers

File: packages/base/Page/src/PageContent/PageContent.tsx [hunk 1]
```diff
@@ -17,34 +15,27 @@ export interface Props extends BaseProps, HTMLAttributes<HTMLDivElement> {
...
 
-    const innerClassName = cx(
-      {
-        [classes.fullWidth]: fullWidth || width === 'full',
-        [classes.wide]: width === 'wide',
-        [classes.flex]: flex,
-      },
-      classes.content
+    const innerClassName = twJoin(
+      'h-full grow bg-[#fcfcfc] bg-clip-content',
```
Comment (dmaklygin) [inline packages/base/Page/src/PageContent/PageContent.tsx:26]: Oh, this is completely new color.
Maybe we should reach Matt to manage this inconsistency... We shouldn't use HEX colors directly

File: packages/base/Page/src/PageContent/PageContent.tsx [hunk 2]
```diff
@@ -17,34 +15,27 @@ export interface Props extends BaseProps, HTMLAttributes<HTMLDivElement> {
...
       <div
         {...rest}
         ref={ref}
-        className={cx(
-          classes.root,
-          { [classes.hasSidebar]: hasSidebar },
+        className={twMerge(
+          'flex flex-[1] w-full justify-center',
+          hasSidebar &&
+            'bg-[linear-gradient(90deg,rgb(243,244,246)50%,rgb(252,252,252)50%)]',
```
Comment (dmaklygin) [inline packages/base/Page/src/PageContent/PageContent.tsx:38]: This line also confuses me. I think, we should reuse existing colors somehow...

Comment (ruslan-sed) [reply packages/base/Page/src/PageContent/PageContent.tsx:38]: @dmaklygin 

Here as in the case of the previous comment the same colors are used as before migration. We can't achieve the same result with different colors. What do you suggest?

Comment (ruslan-sed) [reply packages/base/Page/src/PageContent/PageContent.tsx:38]: We can create a follow-up ticket to handle this after migration.

Comment (ruslan-sed) [reply packages/base/Page/src/PageContent/PageContent.tsx:38]: Considering that Matthew is on the offsite during this week, @TomasSlama suggested to create a follow-up ticket to handle this color situation. Here is a ticket - https://toptal-core.atlassian.net/browse/FX-6029.

**Review by TomasSlama — APPROVED** [review]

**Pull request merged**